docs/layouts/shortcodes/disable-cache.md

```diff
@@ -0,0 +1,14 @@
+
+## Check the routes settings
```

Comment:
:lady_beetle: This heading should be more specific. Right now it's about settings, which isn't something we want to focus on (settings aren't a user goal, just a means to achieve the goal). A heading focused on why they'd want to do the step would be better.

Reply:
I hope it's better now? I don't like the repetition of `avoid`, but it should do the trick.
`double-caching` is probably easier to understand as a headline. Or would you go with `Avoid stale content`?.